src/locutus/__init__.py

```diff
@@ -13,19 +13,59 @@ def strip_none(value):
     return value
 
 
-def fix_varname(varname):
-    return varname.strip().replace(" ", "_")
-
-
-def clean_varname(name):
-    return (
-        name.lower()
-        .replace(" ", "_")
-        .replace("(", "")
-        .replace(")", "")
-        .replace("'", "")
-        .replace('"', "")
-    )
+FTD_PLACEHOLDERS = {
+    "<FTD-DOT>": ".",
+    "<FTD-DOT-DOT>": ".."
+    }
+
+def normalize_ftd_placeholders(code):
+    """
+    Replaces special FTD placeholders if the entire code matches them.
+    
+    Args:
+        code(str): The input code.
+
+    Returns:
+        str: The normalized code.
+
+    """
+    if code in FTD_PLACEHOLDERS:
+        return FTD_PLACEHOLDERS[code]
+    else:
+        return code
+    
+
+# Special character mappings. UTF-8 Hex
+sp_char_mappings_indexes = {'/': '0x2F'}
+
+def get_code_index(code):
+    """
+    Cleans the code identifier, for db path referencing
+
+    Background: Codes from various inputs(request url, sideload, ect.)
+    might contain special characters that cannot be used in a firestore resource
+    path. i.e. `Ontology/Code` Use this function to clean them. Note that
+    Codings(i.e. Table Variables) and CodingMappings(i.e. Mapping objects) 
+    should not have transformed codes, another function exists for those transformations.
+
+    Args:
+      code(str): code. 
+      Examples: `given/code`, `..`, or `<FTD-DOT-DOT>
+
+    Output:
+      code_index(str): 
+      Examples: `given0x2Fcode' or <FTD-DOT-DOT>`
+    """
+
+    # Ensure any codes with designated placeholders have them in place at indexing.
+    REVERSE_FTD_PLACEHOLDERS = {v: k for k, v in FTD_PLACEHOLDERS.items()}

```

Comment:
I would recommend against putting this inside a function, since it will just redo the same work over and over again. Maybe make it a sibling to the actual FTD Placeholders object?